python/cudf/cudf/core/column/decimal.py

```diff
@@ -157,7 +157,13 @@ def normalize_binop_value(self, other):
                 other = other.astype(self.dtype)
             return other
         elif is_scalar(other) and isinstance(other, (int, Decimal)):
-            return cudf.Scalar(Decimal(other), dtype=self.dtype)
+            other = Decimal(other)
+            metadata = other.as_tuple()
+            precision = max(len(metadata.digits), metadata.exponent)
+            scale = -metadata.exponent
+            return cudf.Scalar(
```

Comment:
If `other` is `Decimal` should this not somehow end up going through the same normalisation as above with `cudf.Scalar` with decimal dtypes? Or does it not matter?

I'm pretty shaky on all the decimal stuff though, so ...